service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateSentinelRunPlaybookAutomationRule.java

```diff
@@ -0,0 +1,90 @@
+package bio.terra.landingzone.stairway.flight.create.resource.step;
+
+import bio.terra.landingzone.library.configuration.LandingZoneProtectedDataConfiguration;
+import bio.terra.landingzone.library.landingzones.definition.ArmManagers;
+import bio.terra.landingzone.library.landingzones.definition.ResourceNameGenerator;
+import bio.terra.landingzone.library.landingzones.definition.factories.ParametersResolver;
+import bio.terra.landingzone.service.landingzone.azure.model.LandingZoneResource;
+import bio.terra.landingzone.stairway.flight.exception.MissingRequiredFieldsException;
+import bio.terra.stairway.FlightContext;
+import com.azure.resourcemanager.securityinsights.models.AutomationRuleRunPlaybookAction;
+import com.azure.resourcemanager.securityinsights.models.AutomationRuleTriggeringLogic;
+import com.azure.resourcemanager.securityinsights.models.PlaybookActionProperties;
+import com.azure.resourcemanager.securityinsights.models.TriggersOn;
+import com.azure.resourcemanager.securityinsights.models.TriggersWhen;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CreateSentinelRunPlaybookAutomationRule extends BaseResourceCreateStep {
+  private static final Logger logger =
+      LoggerFactory.getLogger(CreateSentinelRunPlaybookAutomationRule.class);
+
+  private final LandingZoneProtectedDataConfiguration landingZoneProtectedDataConfiguration;
+
+  public CreateSentinelRunPlaybookAutomationRule(
+      ArmManagers armManagers,
+      ParametersResolver parametersResolver,
+      ResourceNameGenerator resourceNameGenerator,
+      LandingZoneProtectedDataConfiguration landingZoneProtectedDataConfiguration) {
+    super(armManagers, parametersResolver, resourceNameGenerator);
+    this.landingZoneProtectedDataConfiguration = landingZoneProtectedDataConfiguration;
+  }
+
+  @Override
+  protected void createResource(FlightContext context, ArmManagers armManagers) {
+    var workspace =
```

Comment:
"workspace" here refers to "logAnalyticsWorkspace"? Could you go with the longer name so it's clear to folks what it is (my brain thinks other things when it sees "workspace:)?